src/ui/src/components/chart/JinjatECharts.tsx

```diff
@@ -1,74 +1,79 @@
 import ReactECharts from "echarts-for-react";
-import React, {useMemo} from "react";
-import {useCustom} from "@refinedev/core";
+import React, { useMemo } from "react";
+import { useCustom } from "@refinedev/core";
 import * as echarts from "echarts";
-import {memoize} from "lodash";
-import {JinjatDataset} from "@components/crud/utils";
+import { memoize } from "lodash";
+import { JinjatDataset } from "@components/crud/utils";
 import PropTypes from "prop-types";
 
 export interface JinjatEChartsProps {
-    dataset: JinjatDataset;
-    options?: EChartsOptions;
-    theme?: string
+  dataset: JinjatDataset;
+  options?: EChartsOptions;
+  theme?: string;
 }
 
 type EChartsOptions = object;
 
 export const JinjatECharts: React.FC<JinjatEChartsProps> = ({
-                                                                dataset,
-                                                                options = {},
-                                                                theme,
-                                                            }) => {
-    // @ts-ignore
-    const isRenderable = memoize(() => (Array.isArray(options?.dataset) && options?.dataset[0]?.source == null) || (options?.dataset?.source == null))
-    const { data, isLoading, isError } = useCustom<[]>({
-        url: `/_analysis/${dataset.analysis}`,
-        method: "get",
-        config: {
-            query: dataset.query_params,
-            payload: dataset.body
-        },
-        queryOptions: {
-            queryKey: ["_analysis", dataset],
-        }
-    });
+  dataset,
+  options = {},
+  theme,
+}) => {
+  // @ts-ignore
+  const isRenderable = memoize(
+    () =>
```

Comment:
please revert the changes in this file as if you didn't change the code in this file.

Reply:
Okay. That was just a formatting edit. Will revert it.